ansible_collections/arista/avd/roles/eos_cli_config_gen/schemas/schema_fragments/router_service_insertion.schema.yml

```diff
@@ -8,7 +8,42 @@ type: dict
 keys:
   router_service_insertion:
     type: dict
-    description: Configure network services inserted to data forwarding
+    description: Configure network services inserted to data forwarding.
     keys:
       enabled:
         type: bool
+      connections:
+        type: list
+        items:
+          type: dict
+          keys:
+            name:
+              type: str
+              description: Configure name of the network service.
+            interface:
+              type: dict
+              description: Interface to be used to reach the network service.
+              keys:
+                ethernet:
+                  type: dict
+                  keys:
+                    interface_name:
+                      type: str
+                      description: e.g. ethernet2
+                    next_hop_ip_address:
+                      type: str
+                      description: Configure the next-hop ip address.
+                tunnel:
+                  type: dict
+                  keys:
+                    interface_name:
+                      type: str
+                      description: e.g. tunnel2
+                    tunnel_config:
+                      type: str
+                      valid_values:
+                        - "primary"
+                        - "secondary"
+            monitor_connectivity_to_host:
```

Comment:
Check if multiple monitor connectivity is possible.

Reply:
Multiple monitor connectivity is not possible.